p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -6130,7 +6139,7 @@ table t {
 The ```@name``` annotation directs the compiler to use a different
 local name when generating the external APIs used to manipulate a
 language element from the control plane. It must have a string literal
-argument, which shall not contain the "```.```" character.  In the
+argument.  In the
```

Comment:
Yes, it is important, a name annotation can still be relative and contain a path. There are absolute paths `@globalname` and relative paths `@name`. But they are paths, not names. The compiler creates paths when inlining, for example.